AntWars/Helper/Utils.cs

```diff
@@ -79,5 +79,28 @@ public static void saveConfigToFile(Config config)
             Utils.xmlSerializer.Serialize(file, config);
             file.Close();
         }
+
+        /// <summary>
+        /// Überprüft ob die Attribute der Ameise in dem vorgegebenem Bereich sind.
+        /// </summary>
+        /// <param name="viewRange">Die Sichtweite der Ameise.</param>
+        /// <param name="moveRange">Die Reichweite der Ameise.</param>
+        /// <param name="inventory">Die Inventargröße der Ameise.</param>
+        public static void checkAttributes(int viewRange, int moveRange, int inventory)
+        {
+            // TODO: Get Min and Max from CostCalculator-Form or from global const
```

Comment:
Vorschlag: eine globale, konstante Klasse in der alle Konstanten liegen, unter anderem auch daten aus der Config.